### PR TITLE
Blockscout integration

### LINKED ISSUE #2469: Integrate BlockScout
### User Story

**As a** user of Gitcoin, 
**I want to** have [BlockScout ](https://poa.network/blockscout) integrated with Gitcoin
**so that** I have  an open-source alternative to Etherscan to view tx details

### Why Is this Needed

It would be great to have many different alternatives to viewing tx information across the Gitcoin platform so that there is not dependency on one single product.

Full disclosure: BlockScout is an ETHPrize bounty recipient and I am the Director of Product for ETHPrize. I have an (open-source, not financial) interest in seeing the product get adopted by the ecosystem at large :)

### Description

Integrated BlockScout where appropriate into the Gitcoin platform. [Here](https://poa.network/blockscout) is the site for BlockScout and here is BlockScout's [Github repo](https://github.com/poanetwork/blockscout).

### Definition of Done

BlockScout is integrated everywhere that Etherscan is currently implemented for Gitcoin.

### PR DESCRIPTION
**Pull request by FrederikBolding**
##### Description

Integrating Blockscout as an alternative where Etherscan is currently implemented.

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [x] Linter status: 100% pass
- [x] Added relevant tests
- [x] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [x] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Refers/Fixes

Fixes #2469 

###### Contributor

- [x] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [x] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

**Comment by FrederikBolding** [comment]
Still WIP, just creating the PR for the sake of transparency, and to track my progress 😄

File: app/economy/tests/test_economy_utils.py [hunk 1]
```diff
@@ -68,4 +68,11 @@ def test_convert_amount_time_travel(self):
...
         txid = '0xcb39900d98fa00de2936d2770ef3bfef2cc289328b068e580dc68b7ac1e2055b'
-        assert etherscan_link(txid) == 'https://etherscan.io/tx/0xcb39900d98fa00de2936d2770ef3bfef2cc289328b068e580dc68b7ac1e2055b'
+        assert etherscan_link(
+            txid) == 'https://etherscan.io/tx/0xcb39900d98fa00de2936d2770ef3bfef2cc289328b068e580dc68b7ac1e2055b'
+
+    def test_blockscout_link(self):
+        """Test the economy util blockscout_link method."""
+        txid = '0xcb39900d98fa00de2936d2770ef3bfef2cc289328b068e580dc68b7ac1e2055b'
+        assert blockscout_link(
+            txid) == 'https://blockscout.com/eth/mainnet/tx/0xcb39900d98fa00de2936d2770ef3bfef2cc289328b068e580dc68b7ac1e2055b'
```
[the remark this answers is not in the document]
Comment (FrederikBolding) [reply app/economy/tests/test_economy_utils.py]: Not exactly sure about what to do with this one.

**Comment by FrederikBolding** [comment]
Would love some feedback on this PR, along with the comments I've made on #2469

**Comment by owocki** [comment]
code LGTM!   pestered mitch to payout!

**Comment by thelostone-mc** [comment]
Closing this out as the we are sticking to etherscan for now and the issue is already closed out ! 
Thanks @FrederikBolding ^_^

**Pull request closed without merging**